src/model.F90

```diff
@@ -33,7 +44,7 @@ subroutine demo_rad()
       do i_time = 1, 5
          ! Initialize the timestep
          call demo_rad_ccpp_physics_timestep_initial('toy_suite',             &
-              errmsg, errcode)
+              aerosol_model_name, errmsg, errcode)
```

Comment:
Why do you need `aerosol_model_name` here (along with at run and final times)? Moving this item from the host table to a module table can fix this.